bash_conditionals_loops/bash_conditionals_loops.md

```diff
@@ -344,7 +355,8 @@ That said, the Bash language does not make it easy to write conditional statemen
 * how to recognize conditionals in Bash
 * how to make small changes to existing code.
 
-<div class = "important">
+<div class = "care">
```

Comment:
not sure this is "care" worthy. ( "The style "care" is for content related to compassion, self-care, and motivation. ") 

maybe these don't need a div box? Or could be presented in the additional resources section rather than mid-module?

Reply:
removed this line, these resources are already linked in the Additional Resources section :)